src/stories/assets/repl.js

```diff
@@ -1,14 +1,19 @@
 export class Repl {
-  constructor (gotLineCallback, element) {
+  constructor (gotLineCallback) {
     this.state = {
       history: [],
       input: ''
     }
 
-    this.gotLineCallback = gotLineCallback
     this.replRoot = {}
+    this.gotLineCallback = gotLineCallback
+
+    this.open()
+  }
+
+  open () {
     this.iframe = document.createElement('iframe')
-    element.appendChild(this.iframe)
+    document.body.appendChild(this.iframe)
```

Comment:
Should this have some styling to be hidden and not take space in the page?
![image](https://user-images.githubusercontent.com/1295829/37345802-63e4aed6-26d6-11e8-964f-e91f4e339295.png)

Reply:
yep good idea